### DISCUSSION TITLE
Disable config preset in a repository

### DISCUSSION THREAD

**Discussion by Stefan-J-Hanke** [renovatebot/renovate#20836] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.152.4

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

15.9.2

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

The global renovate-config contains the preset "group:allNonMajor".
Can this be undone in a renovate config inside a repository in some way?

### What you tried so far.

Well checked the documentation but didn't find anything. Since that preset effectively evaluates to a package rule I'd say it could be overridden by another rule but that's not practical in this case.

We're now investigating the possibility of having multiple configs and tweak our gitlab pipeline setup to use different configs for some GitLab projects.

Perfectly fine with a statement that it won't work - we're just unsure.

**Comment by rarkins** [comment, marked as answer]
Look up `ignorePresets`

**Reply by Stefan-J-Hanke** [reply]
thanks - appreciate your help!
